providers/fab/src/airflow/providers/fab/www/views.py

```diff
@@ -77,22 +75,11 @@ def index(self):
 
 def show_traceback(error):
     """Show Traceback for a given error."""
-    is_logged_in = get_auth_manager().is_logged_in()
     return (
         render_template(
             "airflow/traceback.html",
-            python_version=sys.version.split(" ")[0] if is_logged_in else "redacted",
-            airflow_version=version if is_logged_in else "redacted",
-            hostname=(
-                get_hostname()
-                if conf.getboolean("webserver", "EXPOSE_HOSTNAME") and is_logged_in
-                else "redacted"
-            ),
-            info=(
-                traceback.format_exc()
-                if conf.getboolean("webserver", "EXPOSE_STACKTRACE") and is_logged_in
-                else "Error! Please contact server admin."
-            ),
+            python_version=sys.version.split(" ")[0],
```

Comment:
Maybe just remove this too? Less is more.

Reply:
I thought about it but yes you're right

Reply:
Updated